scripts/crawl.py

```diff
@@ -0,0 +1,86 @@
+import bs4
+import json
+import requests
+from pathlib import Path
+from typing import Union
+from argparse import ArgumentParser
```

Comment:
Please sort the imports alphabetically. Also, observe [the PEP-8 rules for imports](https://peps.python.org/pep-0008/#imports).